api/README.md

```diff
@@ -303,6 +315,33 @@ Content-Type: application/json; charset=utf-8
 ```
 
 
+## [DEPRECATED] GET /api/v1/export/forms/{formcode}
```

Comment:
Not sure why github thinks this is new text 🤷‍♂️

Reply:
Yeah the diff on this PR is atrocious, sorry